packages/trader/src/Modules/Trading/Components/Form/TradeParams/Accumulator/accumulator.jsx

```diff
@@ -8,21 +8,21 @@ import { connect } from 'Stores/connect';
 import { getGrowthRatePercentage, getTickSizeBarrierPercentage } from '@deriv/shared';
 import classNames from 'classnames';
 
-const Accumulator = ({ accumulator_range_list, growth_rate, onChange, tick_size_barrier }) => {
+const Accumulator = ({ accumulator_range_list, growth_rate, onChange, tick_size_barrier, amount }) => {
     // splitting accumulator_range_list into rows containing 5 values each:
     const arr_arr_numbers = accumulator_range_list.reduce((acc, _el, index) => {
         if (index % 5 === 0) {
             acc.push(accumulator_range_list.slice(index, index + 5));
         }
         return acc;
     }, []);
-
     if (!accumulator_range_list.length) return null;
     return (
         <Fieldset
             className={classNames('trade-container__fieldset', 'accumulator')}
             header={localize('Accumulate')}
             is_center
+            is_tooltip_disabled={amount < 1 || amount > 1000}
```

Comment:
@henry-deriv unfortunately, we cannot have hard-coded values here because min stake and max stake limits are coming from BE. Let's rely on having an error in proposal response instead.
So we can use the following condition for identifying an error like in we do in contract-info to disable the icon based on this:
`const has_error_or_not_loaded = proposal_info.has_error || !proposal_info.id`
proposal_info can be taken from trade-store👍🏻